pkg/agent/cniserver/ipam/ipam_delegator.go

```diff
@@ -26,7 +26,8 @@ import (
 )
 
 const (
-	IPAM_HOST_LOCAL = "host-local"
+	IPAM_HOST_LOCAL  = "host-local"
+	DEFAULT_CNI_PATH = "/opt/cni/bin"
```

Comment:
Not introduced by this patch, should we use Go naming style: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps?

Reply:
Sure. Fixed it.